versioned_docs/version-2.0/concepts/admin/admin.mdx

```diff
@@ -0,0 +1,45 @@
+# Admin
+
+This conceptual guide covers topics related to managing users, organizations, and workspaces within LangSmith.
+
+## Organizations
+
+An organization is a logical grouping of users within LangSmith. Typically, there is one organization per company. An organization can have multiple workspaces.
+
+## Workspaces
+
+A workspace is a logical grouping of users within an organization that share access to the same set of resources, including tracing projects, datasets, annotation queues, and prompts.
```

Comment:
nit: to make it very explicit that the resources are also tied to an individual workspace, maybe something like this:
> A workspace is a logical grouping of users and resources within an organization. Users may have permissions in a workspace that grant them access to the resources in that workspace, including tracing projects, datasets, annotation queues, and prompts.